pkg/ui/src/views/devtools/containers/raftMessages/index.tsx

```diff
@@ -81,7 +81,7 @@ export class RaftMessages extends React.Component<RaftMessagesProps> {
     this.setClusterPath(selected.value);
   }
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
I know these are good changes to make but if you want to address React upgrade warnings, put them in a separate PR.